### ISSUE TITLE
Cannot reproduce

### ISSUE THREAD

**Issue by xiaoye-hhh** [lhf12278/IBAN#2]
Nice work！I run the code, but I only get a lower performance than results in the paper.
![image](https://user-images.githubusercontent.com/55238145/232940087-3273ef6d-310f-41fa-b9a2-6536fc9dec5e.png)
I just run with:
```
python train.py
```
How can I improve the performance?

**Comment by liu199712** [comment]
The code provided  does not have hyperparameters in loss function. You can load the model we trained before, and you can get the same results.

**Comment by LiuMing-Hui** [comment]
> Nice work！I run the code, but I only get a lower performance than results in the paper. ![image](https://user-images.githubusercontent.com/55238145/232940087-3273ef6d-310f-41fa-b9a2-6536fc9dec5e.png) I just run with:
> 
> ```
> python train.py
> ```
> 
> How can I improve the performance?

The code provided does not have hyperparameters in loss function. You can load the model we trained before, and you can get the same results.

**Comment by YoungKyle** [comment]
> > Nice work！I run the code, but I only get a lower performance than results in the paper. ![image](https://user-images.githubusercontent.com/55238145/232940087-3273ef6d-310f-41fa-b9a2-6536fc9dec5e.png) I just run with:
> > ```
> > python train.py
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > How can I improve the performance?
> 
> The code provided does not have hyperparameters in loss function. You can load the model we trained before, and you can get the same results.

请问可以重新提供以下权重吗？网页找不到了。

**Comment by xyz999-skech** [comment]
> > Nice work！I run the code, but I only get a lower performance than results in the paper. ![image](https://user-images.githubusercontent.com/55238145/232940087-3273ef6d-310f-41fa-b9a2-6536fc9dec5e.png) I just run with:
> > ```
> > python train.py
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > How can I improve the performance?
> 
> The code provided does not have hyperparameters in loss function. You can load the model we trained before, and you can get the same results.

请问可以重新提供以下权重吗？网页找不到了。

**Comment by LiuMing-Hui** [comment]
> > > Nice work！I run the code, but I only get a lower performance than results in the paper. ![image](https://user-images.githubusercontent.com/55238145/232940087-3273ef6d-310f-41fa-b9a2-6536fc9dec5e.png) I just run with:
> > > ```
> > > python train.py
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > How can I improve the performance?
> > 
> > 
> > The code provided does not have hyperparameters in loss function. You can load the model we trained before, and you can get the same results.
> 
> 请问可以重新提供以下权重吗？网页找不到了。

可以的，请加我微信liumh127

**Comment by LiuMing-Hui** [comment]
可以的，请加微信liumh127
LiuMinghui
---- Replied Message ----
***@***.***>Date05/23/2024 17:08 ***@***.***> ***@***.***>,
***@***.***>SubjectRe: [lhf12278/IBAN] Cannot reproduce (Issue #2)
Nice work！I run the code, but I only get a lower performance than results in the paper. I just run with:
python train.py 
How can I improve the performance?
The code provided does not have hyperparameters in loss function. You can load the model we trained before, and you can get the same results.
请问可以重新提供以下权重吗？网页找不到了。
